EIPS/eip-7525.md

```diff
@@ -0,0 +1,161 @@
+---
+eip: 7525
```

Comment:
number will be assigned by editors

Reply:
Thank you. Should I leave it empty or assign it as 'tmp' before the editor takes care of it?

Reply:
@abcoathup generally assigns, just wait for him to respond and update :)

Reply:
7525